Mathlib/Data/ENNReal/Operations.lean

```diff
@@ -184,11 +184,11 @@ section OperationsAndInfty
 
 variable {α : Type*}
 
-@[simp] theorem add_eq_top : a + b = ∞ ↔ a = ∞ ∨ b = ∞ := WithTop.add_eq_top
-#align ennreal.add_eq_top ENNReal.add_eq_top
+-- @[simp] theorem add_eq_top : a + b = ∞ ↔ a = ∞ ∨ b = ∞ := WithTop.add_eq_top
+-- #align ennreal.add_eq_top ENNReal.add_eq_top
```

Comment:
Can't you align to the general lemma instead?
```suggestion
#align ennreal.add_eq_top add_eq_top -- or however it's called
```

Reply:
Done.